### ISSUE TITLE
Fix spurious indent violation when a comment follows the end of a multi-line braced word

### ISSUE THREAD

**Issue by nmoroze** [nmoroze/tclint#48]
E.g.

```tcl
puts {
  Hello} ;# asdf
```

Expects no violations, gets

```
example.tcl:2:1: expected indent of 0 spaces, got 2 [indent]
```

**Comment by nmoroze** [comment]
Fixed by 2ffdbdc717f726fa97e2a2ce4b4777a31189b8ec
